flake.nix

```diff
@@ -34,12 +57,8 @@
       # for backward compatibility, is safe to delete, not referenced anywhere
       overlay = self.overlays.default;
 
-      # Run Hercules CI for these systems.
-      herculesCI.ciSystems = [ "x86_64-linux" ];
-
-    } // flake-utils.lib.eachSystem [ "aarch64-linux" "x86_64-linux" ] (system:
-    {
       hydraJobs =
+        lib.genAttrs [ "x86_64-linux" ] (system:
```

Comment:
`aarch64-linux` is removed from `hydraJobs`.  Is it intended?

Reply:
Nope :)